Dudas Dorian/Lab4/5_3.5.c

```diff
@@ -0,0 +1,105 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    int rows,cols;
+    printf("give a value for the rows and collumns: ");
+    scanf("%d",&rows);
+    cols=rows;
+
+    int**B=(int*)malloc(sizeof(int*)*rows);
+    for(int i=0; i<rows; i++)
+    {
+        B[i]=(int*)malloc(sizeof(int)*cols);
+    }
+
+    for(int i=0; i<rows; i++)
+    {
+        for(int j=0; j<cols; j++)
+        {
+            printf("B[%d][%d]=",i,j);
+            scanf("%d",&B[i][j]);
+        }
+    }
+
+    int**C=(int*)malloc(sizeof(int*)*rows);
+    for(int i=0; i<rows; i++)
+    {
+        C[i]=(int*)malloc(sizeof(int)*cols);
+    }
+
+    for(int i=0; i<rows; i++)
+    {
+        for(int j=0; j<cols; j++)
+        {
+            printf("C[%d][%d]=",i,j);
+            scanf("%d",&C[i][j]);
+        }
+    }
+
+    int**ADD=(int*)malloc(sizeof(int*)*rows);
```

Comment:
WHY ARE YOU SCREAMING WITH ALL CAPS VARIABLES

pretty please don't, unless they're global variables/constants